app/controllers/passwordless/sessions_controller.rb

```diff
@@ -65,6 +70,14 @@ def destroy
       redirect_to passwordless_sign_out_redirect_path
     end
 
+    def set_role
```

Comment:
Need to ensure that the role ids never change.

Reply:
True.  Any idea on a better way to do this?

Reply:
Actually I don't think this is a problem.  RoleIDs shouldn't change - if there is a requirement to disable roles later, that should be a separate concern.